### ISSUE TITLE
Sync requires first edit to be made in Sublime Text, using DevTools first doesn't work

### ISSUE THREAD

**Issue by lrobeson** [livestyle/chrome#23]
In order for my changes to be synced/bi-directional, I have to initiate the connection between Sublime Text and Chrome by making some sort of edit in my ST file first. I can't start out in DevTools, if that makes sense. Is this intended behavior?

**Comment by sergeche** [comment]
No, it’s an issue. I’m working on a new Chrome extension plugin right now (which should also improve workflow with preprocessors), looks like it was fixed in new version
